account_journal_lock_date/models/account_journal.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2017 ACSONE SA/NV
+# Copyright 2018 XOE Corp. SAS <https://xoe.solutions>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import timedelta
+
+from odoo import api, fields, models, _
+from odoo.tools.misc import DEFAULT_SERVER_DATE_FORMAT
+from odoo.exceptions import ValidationError
+
+
+class AccountJournal(models.Model):
+
+    _inherit = 'account.journal'
+
+    journal_lock_date_period_offset = fields.Integer(
+        string="Lock date offset", default=0,
```

Comment:
@blaggacao no need to `default = 0`

https://odoo-new-api-guide-line.readthedocs.io/en/latest/fields.html#integer